parsers/round-number/README.md

```diff
@@ -0,0 +1,348 @@
+# Round number
+
+## Description
+
+Rounds any measurement with specific precision. 
+You can easily round any measurement value (font size, line height, blur etc.) and specify the number of decimals.
+By specifying a mode, you can control the round function.
+
+## Interface
+
+```ts
+interface parser {
+  "name": "round-number",
+  "options": {
+    keys: Array<string>;
+    precision?: number;
+    mode?: 'down' | 'up' | 'auto';
+  }
+}
+```
+
+### Options
+
+| Parameter              | Required   | Type             | Default    | Description                                                                                                                |
+| ---------------------- | ---------- | --------------   | ---------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `keys`                 | required   | `Array<string>`  |            | List of key path to apply the rounding |
+| `precision`            | optional   | `number`         | `2`        | Number of decimal expected   |
+| `mode`                 | optional   | `down, up, auto` | `auto`     | up :  rounds a number up to the next largest number with specific decimals </br></br> down :  rounds a number down to the nearest less or equal number with specific decimals </br></br> auto: returns the value of a number rounded to the nearest number with specific decimals |
+
+## Types
+
+ℹ️ **Please be aware that, depending on the order you use parsers, their input and output types have to match.**
+### Input
+
+
+```ts
+Array<Record<string, any>>
+```
+
+### Output
+
+```ts
+Array<Record<string, any>>
+```
+
+## Basic Usage
+### Config
+
+```json
+{
+  "name": "round-number",
+  "options": {
+    "keys": ["value.measure"],
+    "precision": 1,
+    "mode": "auto"
+  }
+}
+```
+
+With this config, we will round the measure of a measurement token 
+### Before/After
+
+#### Input
+
+```json5
+[
+  {
+    "type": "measurement",
+    "value": {
+      "unit": "px",
+      "measure": 13.84  // <---
+    },
+    "name": "size-01"
+  },
+  {
+    "type": "measurement",
+    "value": {
+      "unit": "px",
+      "measure": 16.4 // <---
+    },
+    "name": "size-02"
+  }
+]
+```
+#### Output
+
+```json5
+[
+  {
+    "type": "measurement",
+    "value": {
+      "unit": "px",
+      "measure": 13.8 // <---
```

Comment:
In a web based environment 13.8px is almost an error. We should round the value to its nearest value. In this case I would rather display `14` instead of` 13.8`.